devlib/module/cpufreq.py

```diff
@@ -414,10 +414,12 @@ def trace_frequencies(self):
     def get_domain_cpus(self, cpu):
         """
         Get the CPUs that share a frequency domain with the given CPU
+
+        Returned as a tuple of CPU IDs
         """
         if isinstance(cpu, int):
             cpu = 'cpu{}'.format(cpu)
 
         sysfile = '/sys/devices/system/cpu/{}/cpufreq/affected_cpus'.format(cpu)
 
-        return [int(c) for c in self.target.read_value(sysfile).split()]
+        return tuple(int(c) for c in self.target.read_value(sysfile).split())
```

Comment:
Fair enough. I thought this was necessary for the following commit (because of the use of `set`) but I just realised that was a brainfart. Will just drop this commit.